### ISSUE TITLE
Use CACHE_DIR for composer packages

### ISSUE THREAD

**Issue by henrikbjorn** [facebookarchive/heroku-buildpack-hhvm#17]
Currently ALL packages are redownloaded on every build, this means it takes a lot longer to deploy simple changes. To overcome this it would be beneficial to use CACHE_DIR and cache dependencies before installing, also the composer binary could be cached there.

**Comment by ClaudiuC** [comment]
Yeah, I actually had a note to fix this and never got around to do it :+1: 

Feel free to send a PR, I'll try to get to it later this week otherwise.

**Comment by henrikbjorn** [comment]
@ClaudiuC does the same contribution rules apply for this as for hhvm? Just saw the Contributor License Agreement etc.

**Comment by ClaudiuC** [comment]
I think you just need to fill that in once. I was under the impression you already did though. @ptarjan ?

**Comment by ptarjan** [comment]
We aren't going to need a CLA for this repository. I'll add a BSD licence file.

**Comment by ClaudiuC** [comment]
Oh great, thanks for the heads up!
